host/4/bookworm/powershell/powershell74/powershell74-core-tools.Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# This Dockerfile adds a non-root user with sudo access. Use the "remoteUser"
+# property in devcontainer.json to use it. On Linux, the container user's GID/UIDs
+# will be updated to match your local UID/GID (when using the dockerFile property).
+# See https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+ARG PS_VERSION=7.4.0-rc.1
+ARG PS_PACKAGE=powershell-preview_${PS_VERSION}-1.deb_amd64.deb
```

Comment:
@CooperLink here, the link to pull the PowerShell package has preview in the title. Compare with the 7.2 image for what it looks like for a release PS version